internal/services/compute/virtual_machine_scale_set_extension_resource_test.go

```diff
@@ -59,12 +59,12 @@ func TestAccVirtualMachineScaleSetExtension_requiresImport(t *testing.T) {
 	})
 }
 
-func TestAccVirtualMachineScaleSetExtension_autoUpgradeDisabled(t *testing.T) {
+func TestAccVirtualMachineScaleSetExtension_autoUpgradeMinorVersionDisabled(t *testing.T) {
```

Comment:
renamed test do distinguish between `auto_upgrade_minor_version` and `automatic_upgrade_enabled`